sdk/python/jobs/automl-standalone-jobs/automl-nlp-text-named-entity-recognition-task-distributed-sweeping/automl-nlp-text-ner-task-distributed-with-sweeping.ipynb

```diff
@@ -460,6 +472,59 @@
    ]
   },
   {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.3 Manual hyperparameter sweeping for models from Hugging Face (Preview)\n",
+    "Similar to how you can use any model algorithm from Hugging face transformers library for individual runs, you can also include these models to perform a hyperparameter sweep. You can also choose a combination of model algorithms supported supported natively by [AutoML](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=cli) and model algorithms from [Hugging Face](https://huggingface.co/models?pipeline_tag=token-classification&library=transformers&sort=trending).\n",
+    "\n",
+    "In this example, we sweep over bert-base-cased, microsoft/xdoc-base-funsd, and xlm-roberta-large-finetuned-conll03-english, models choosing from a range of values for learning_rate, number_of_epochs, etc., to generate a model with the optimal 'accuracy'."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the AutoML job with the related factory-function.\n",
+    "\n",
+    "text_ner_job = automl.text_ner(\n",
+    "    compute=compute_name,\n",
+    "    # name=\"dpv2-text-ner-job-02\",\n",
+    "    experiment_name=exp_name,\n",
+    "    training_data=my_training_data_input,\n",
+    "    validation_data=my_validation_data_input,\n",
+    "    tags={\"my_custom_tag\": \"My custom value\"},\n",
+    ")\n",
+    "\n",
+    "text_ner_job.set_limits(\n",
+    "    timeout_minutes=120, max_trials=4, max_concurrent_trials=2, max_nodes=4\n",
+    ")\n",
+    "\n",
+    "text_ner_job.extend_search_space(\n",
+    "    [\n",
+    "        SearchSpace(\n",
+    "            model_name=Choice([\"bert-base-cased\", \"microsoft/xdoc-base-funsd\"]),\n",
```

Comment:
Are the models "microsoft/xdoc-base-funsd" and "xlm-roberta-large-finetuned-conll03-english" part of the curated models list ? It might be better to add models from curated list [i.e that are exposed in registry] in the samples. Even though, we don't download them from registry, these are well-tested models.

Reply:
Same comment for HF models used in all the notebooks

Reply:
Updated to use model from registry.

Reply:
The models in this search space are still non-registry models.

Reply:
Updated again.